docs/cli.rst

```diff
@@ -69,6 +69,30 @@ See :doc:`data-process` for detail.
    Reference value for date-time substitutions.
    See also :envvar:`YP_TIME_REF_VALUE`.
 
+yp-preprocesor
+--------------
+
+Usage:
+
+.. code-block:: bash
+
+   yp-preprocesor [options] -o output-file-name input-file-name
```

Comment:
Shouls this be:
` yp-preprocessor -i  input-file-name -o output-file-name [options]`

Reply:
changed